spec/controllers/submissions_controller_spec.rb

```diff
@@ -6,7 +6,8 @@
   render_views
```

Comment:
This line is probably not needed as it is written a second time in line 280 for the 'GET #show.json' tests. For me locally the tests pass without it.